From 2d913ddb1dfb1069425f4e7257224af6a5535f10 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Sun, 27 Aug 2023 12:11:19 -0500
Subject: [PATCH] Skip test that fails in chroot

This test runs fine when ran on an actual Solus system

---
 tests/testsuite | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tests/testsuite b/tests/testsuite
index b407c9e..45d6a40 100755
--- a/tests/testsuite
+++ b/tests/testsuite
@@ -731,7 +731,6 @@ at_help_all="1;version.at:19;tar version;;
 119;listed01.at:26;--listed for individual files;listed incremental listed01;
 120;listed02.at:28;working --listed;listed incremental listed02;
 121;listed03.at:24;incremental dump when the parent directory is unreadable;listed incremental listed03;
-122;listed04.at:26;--listed-incremental and --one-file-system;listed incremental listed04;
 123;listed05.at:33;--listed-incremental and remounted directories;listed incremental listed05 chdir;
 124;incr03.at:28;renamed files in incrementals;incremental incr03 rename;
 125;incr04.at:29;proper icontents initialization;incremental incr04 icontents;
-- 
2.42.0

